lib/src/main/java/growthbook/sdk/java/Feature.java

```diff
@@ -1,12 +1,6 @@
 package growthbook.sdk.java;
 
-import com.google.gson.JsonArray;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonObject;
-import com.google.gson.reflect.TypeToken;
```

Comment:
All the JSON serialization stuff was not being used in the actual implementation, only in tests, so I've removed it since we convert to/from JSON with the automated serialization with Gson.